### DISCUSSION TITLE
Only find files

### DISCUSSION THREAD

**Discussion by lauccode** [nvim-telescope/telescope.nvim#1361] [Q&A]
Sorry, but I don't find how to find only files when using 

> nnoremap <leader>ff <cmd>Telescope find_files<cr>

I would like to find only files and not some strings in the path ... 
If someone know how to do ?

**Comment by rodamaral** [comment]
AFAIK, there's no way to do it at the moment. `find_files` is an "one shot job", meaning it calls `fd` without any filename argument to retrieve all possible files and then filters the results using the desired fuzzy finder. It's not like `live_grep` which calls grep/rip_grep on every new input.

The work around is to display only the filename.extension in this picker, but that would suck even more.
